tests/test_main.rs

```diff
@@ -89,3 +90,37 @@ fn no_package_manifest_error() {
     dbg!(&expected_path);
     assert!(stderr.contains(&expected_path));
 }
+
+#[test]
+fn list_script_names() {
+    let temp_dir = tempdir().unwrap();
+    fs::write(
+        temp_dir.path().join("package.json"),
+        include_str!("fixtures/list-script-names.package.json"),
+    )
```

Comment:
do we need to write to a new temp directory if the test doesn't change anything in the package.json?

Reply:
The test itself doesn't have side effects, but `pn` itself may have. We should take into account where some future `pn` bugs could cause unwanted side effects.